testutil/network/network.go

```diff
@@ -175,6 +175,10 @@ func MinimumAppConfig() depinject.Config {
 }
 
 func DefaultConfigWithAppConfig(appConfig depinject.Config) (Config, error) {
+	return DefaultConfigWithAppConfigWithQueryGasLimit(appConfig, 0)
+}
+
+func DefaultConfigWithAppConfigWithQueryGasLimit(appConfig depinject.Config, queryGasLimit uint64) (Config, error) {
```

Comment:
can we godoc this function and `DefaultConfigWithAppConfig`

`DefaultConfigWithAppConfig` is an app config with an infinite gas limit on querys

Reply:
would also be nice to indicate what the zero queryGasLimit implies (infinite?)